pwa/src/components/objectEntities/objectEntityFormNew/ObjectEntityFormNew.tsx

```diff
@@ -0,0 +1,262 @@
+              </button>
+            </Link>
+          </>
+        )
+      }}
+      cardBody={function () {
+        return (
+          <>
+            {
+              showSpinner && <Spinner />
+            } 
+            {
+              !showSpinner && formIO && formIO
+            }
+            {
+              !showSpinner && object && 
+                  <Accordion
+                    id="advancedConfigurationAccordion"
+                  items={[
+                    {
+                      title: "Advanced configuration",
+                      id: "advConfigAccordion",
+                      render: function () {
+                        return (<>
+                          <table className="mt-3 configurationTable-table">
+                            <tr key="URI">
+                              <th>URI</th>
+                              <td>{object?.uri ?? 'No URI found'}</td>
+                            </tr>
+                            <tr key="ExternalID">
+                              <th>External ID</th>
+                              <td>{object.externalId ?? 'No external ID found'}</td>
+                            </tr>
+                            <tr key="Application">
+                              <th>Application</th>
+                              <td>
+                                {
+                                  object.application ?
+                                  <Link to={`/applications/${object.application.id}`}>{object.application.name}</Link>
+                                  : 'No application found'
+                                }
+                              </td>
+                            </tr>
+                            <tr key="Organization">
+                              <th>Organization</th>
+                              <td>{object.organization ?? 'No organization found'}</td>
+                            </tr>
+                            <tr key="Owner">
+                              <th>Owner</th>
+                              <td>{object.owner ?? 'No owner found'}</td>
+                            </tr>
+                          </table>
+      
+                          <Accordion
+                            id="objectAccordion"
+                            items={[
+                              {
+                                title: "Errors",
+                                id: "errorsAccordion",
+                                render: function () {
+                                  return (<h6>IN PROGRESS: needs refinement, no info on how to show data</h6>);
+                                },
+                              },
+                              {
+                                title: "Promises",
+                                id: "promisesAccordion",
+                                render: function () {
+                                  return (<h6>IN PROGRESS: needs refinement, no info on how to show data</h6>);
```

Comment:
I don't think this should be a `<h>` tag. It's usually used to bring structure (and hierarchy) to e.g. articles. And when they're used, they should have a clear hierarchy (so top-level `1`, second-level `2`, and so on).

However, because this is not any kind of article, it's probably best to just keep this as a `<span>` or something along those lines. 

Also, regarding the hierarchy: this `6` is now rendered directly below a `2` (the title of the accordion).